### PR TITLE
Add assets support for FreeBSD

### PR DESCRIPTION
**Pull request by amontalban**
Currently the assets scripts are not working with FreeBSD, so I'm adding support for it.

### CODE REVIEW

**Comment by rickh563** [comment]
ZD-2825

File: assets/install.sh [hunk 1]
```diff
@@ -6,6 +6,7 @@
 ## for kitchen tests to run
 ##
 
+os=`uname -s`
```
Comment (gtmanfred) [inline assets/install.sh]: This should be a subshell, and not backtics

`os="$(uname -s)"

File: assets/install_with_git.sh [hunk 2]
```diff
@@ -6,6 +6,7 @@
 ## for kitchen tests to run
 ##
 
+os=`uname -s`
```
Comment (gtmanfred) [inline assets/install_with_git.sh]: same here

**Review by gtmanfred — CHANGES_REQUESTED** [review]
this looks good, just some small change requests

**Comment by amontalban** [comment]
@gtmanfred pushed.

PS: Do you want me to do a PR to remove the other backtics in this scripts?

**Comment by gtmanfred** [comment]
It also needs to be quoted, to incase of expansion.

Also yes, that would be great if you could remove the other backtics!

I haven't had time to go through and clean up all the asset scripts, but the added tests are here if you want to start with these shellcheck commands https://github.com/saltstack/kitchen-salt/pull/160

**Comment by amontalban** [comment]
@gtmanfred cool, I will do a PR to fix #160 then. If you can merge this now would be great. Thanks!

**Review by gtmanfred — APPROVED** [review]

**Pull request merged**